### PR TITLE
Move Account and MobileClient from py2 to py3

### PR DESCRIPTION
**Pull request by ZachOrr**
Moving Account and MobileClient in preparation for moving TBANS code to test tasks/queues

### CODE REVIEW

File: src/backend/common/queries/mobile_client_query.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+from typing import List
+
+
+class MobileClientListQuery(DatabaseQuery[List[MobileClient]]):
+    @ndb.tasklet
+    def _query_async(
+        self, users: List[str], client_types: List[ClientType] = list(ClientType)
+    ) -> TypedFuture[List[MobileClient]]:
+        if not users or not client_types:
+            return []
```
Comment (phil-lopreiato) [inline src/backend/common/queries/mobile_client_query.py:15]: I have a fix for `pyre` complaining about this, will be up shortly

File: src/backend/common/queries/mobile_client_query.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+        client_types = [c.value for c in client_types]
```
Comment (phil-lopreiato) [inline src/backend/common/queries/mobile_client_query.py:17]: Is this needed? It may need to be a separate variable?

Comment (ZachOrr) [reply src/backend/common/queries/mobile_client_query.py:17]: TIL apparently not 😲

Comment (phil-lopreiato) [reply src/backend/common/queries/mobile_client_query.py]: yeah, since `IntEnum` extends `int`, it should be implicitly convertable

**Pull request merged**